test/src/main/java/org/apache/accumulo/test/util/Wait.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.test.util;
+
+import java.util.concurrent.TimeUnit;
+
+public class Wait {
+
+  public static final long MAX_WAIT_MILLIS = 30 * 1000;
```

Comment:
Could use the `TimeUnit` conversion for these instead of manually calculating seconds to millis.
```suggestion
  public static final long MAX_WAIT_MILLIS = SECONDS.toMillis(30);
```

Reply:
I changed this and fixed the typo in the latest update.